### PR TITLE
Initial revision of conformance node environment conformance tests

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: test/e2e_node/environment/conformance.go [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+			errs = append(errs, errors.New(fmt.Sprintf("Unrecognized option %s", c)))
+		}
+	}
+	return errs
+}
+
+const dockerVersionRegex = `1\.[7-9]\.[0-9]+`
+
+// containerruntime checks that a suitable containerruntime is installed and recognized by cadvisor: docker 1.7-1.9
+func containerruntime() error {
```
Comment (derekwaynecarr) [inline test/e2e_node/environment/conformance.go]: camelCase?

File: test/e2e_node/environment/conformance.go [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+func daemons() error {
+	if exec.Command("pgrep", "-f", "kubelet").Run() != nil {
+		return printError("Deamon Check %s: kubelet process not found", failed)
+	}
+
+	if exec.Command("pgrep", "-f", "kube-proxy").Run() != nil {
+		return printError("Deamon Check %s: kube-proxy process not found", failed)
+	}
+
+	if exec.Command("pgrep", "-f", "docker").Run() != nil {
```
Comment (derekwaynecarr) [inline test/e2e_node/environment/conformance.go]: should docker be checked separately since other runtimes are possible here?

Comment (pwittrock) [inline test/e2e_node/environment/conformance.go]: Yeah, this is probably sufficiently covered by the containerRuntime method for now.

File: test/e2e_node/environment/conformance.go (hunk 1 above)
Comment (pwittrock) [inline test/e2e_node/environment/conformance.go]: Done

**Comment by vishh** [comment]
LGTM. We can iterate on this in the future.

**Pull request merged**